__tests__/api.test.js

```diff
@@ -35,4 +36,40 @@ describe('GET /api', () => {
     })
 })
 
-  
\ No newline at end of file
+describe('GET /api/articles', () => {
+    test('GET 200: should return an array of articles', () => {
+        return request(app).get("/api/articles").expect(200)
+        .then((response) => {
+            const {articles} = response.body
+            expect(articles).toBeInstanceOf(Array)
+            articles.forEach(article => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
+            })
+        })
+    })
+  })
+  test('GET 200: the article should be sorted by date in descending order', () => {
+    return request(app).get("/api/articles").expect(200)
+    .then(({body})=> {
+        const articles = body.articles
+        expect(articles).toBeSortedBy("created_at", {descending:true})
+    })
+  })
+   test('GET 200: the article should not return the body property on the article objects', () => {
+    return request(app).get("/api/articles").expect(200)
+    .then(({body}) => {
+        const articles = body.articles
+        console.log(articles)
+        articles.forEach((article) => {
+            expect(article).not.toHaveProperty('body')
+})
+    })
```

Comment:
I like that you've explicitly got something to check the `body` property does not exist. It might be worth just putting this on to your first test as a final assertion, rather than a separate test block?